dandiapi/api/management/commands/correct_metadata.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import annotations
+
+from copy import deepcopy
+from typing import TYPE_CHECKING, Any
+
+if TYPE_CHECKING:
+    from collections.abc import Callable
+
+from django.db import transaction
+import djclick as click
+
+from dandiapi.api.models import Version
+from dandiapi.api.tasks import write_manifest_files
+
+
+@click.command(
+    help='Correct corrupted metadata. If `--dandiset` and `--dandiset-version` are provided, '
+    'apply the correction to the specified Dandiset version. Otherwise, apply the correction to '
+    'all Dandiset versions.'
```

Comment:
I'd like to reverse the default behavior: by default, require a Dandiset and version; to override, include a flag `--all` that would apply the change to all versions.

Reply:
Done.